[BUG] Securityadmin error:  Not a valid protocol version: This is not an HTTP port.

**What is the bug?**
Running `/usr/share/opensearch/plugins/opensearch-security/tools/securityadmin.sh` on port `9300` throws error `Caused by: org.apache.http.ParseException: Not a valid protocol version: This is not an HTTP port`
Full Error log:
```
**************************************************************************
** This tool will be deprecated in the next major release of OpenSearch **
** https://github.com/opensearch-project/security/issues/1755           **
**************************************************************************
Security Admin v7
Will connect to my-first-cluster.default.svc.cluster.local:9300 ... done
ERR: An unexpected RuntimeException occured: error while performing request
Trace:
java.lang.RuntimeException: error while performing request
	at org.opensearch.client.RestClient.extractAndWrapCause(RestClient.java:932)
	at org.opensearch.client.RestClient.performRequest(RestClient.java:307)
	at org.opensearch.client.RestClient.performRequest(RestClient.java:295)
	at org.opensearch.security.tools.SecurityAdmin.execute(SecurityAdmin.java:464)
	at org.opensearch.security.tools.SecurityAdmin.main(SecurityAdmin.java:161)
Caused by: org.apache.http.ProtocolException: Not a valid protocol version: This is not an HTTP port
	at org.apache.http.impl.nio.codecs.AbstractMessageParser.parse(AbstractMessageParser.java:209)
	at org.apache.http.impl.nio.DefaultNHttpClientConnection.consumeInput(DefaultNHttpClientConnection.java:245)
	at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:81)
	at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:39)
	at org.apache.http.impl.nio.reactor.AbstractIODispatch.inputReady(AbstractIODispatch.java:121)
	at org.apache.http.impl.nio.reactor.BaseIOReactor.readable(BaseIOReactor.java:162)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvent(AbstractIOReactor.java:337)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvents(AbstractIOReactor.java:315)
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.execute(AbstractIOReactor.java:276)
	at org.apache.http.impl.nio.reactor.BaseIOReactor.execute(BaseIOReactor.java:104)
	at org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor$Worker.run(AbstractMultiworkerIOReactor.java:591)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.apache.http.ParseException: Not a valid protocol version: This is not an HTTP port
	at org.apache.http.message.BasicLineParser.parseProtocolVersion(BasicLineParser.java:148)
	at org.apache.http.message.BasicLineParser.parseStatusLine(BasicLineParser.java:366)
	at org.apache.http.impl.nio.codecs.DefaultHttpResponseParser.createMessage(DefaultHttpResponseParser.java:112)
	at org.apache.http.impl.nio.codecs.DefaultHttpResponseParser.createMessage(DefaultHttpResponseParser.java:50)
	at org.apache.http.impl.nio.codecs.AbstractMessageParser.parseHeadLine(AbstractMessageParser.java:156)
	at org.apache.http.impl.nio.codecs.AbstractMessageParser.parse(AbstractMessageParser.java:207)
	... 11 more
```
From 2.0 release notes I see Transport client is deprecated, however the [documentation](https://opensearch.org/docs/latest/security-plugin/configuration/security-admin/) still says for `-p`

`-p | OpenSearch port. Default is 9300—not the HTTP port.`

**How can one reproduce the bug?**
Executing `/usr/share/opensearch/plugins/opensearch-security/tools/securityadmin.sh` on port `9300`

**What is the expected behavior?**
For 1.3.3, this error wont show up and `securityadmin.sh` executes without any error, connects to cluster and creates a security index, works as expected. 

**What is your host/environment?**
 - OS: 2.0.1

**Do you have any additional context?**
Following the issue 
https://github.com/opensearch-project/security/issues/1578, I see the `TransportClient` is deprecated, so ideally the expected port is rest http port, could be even 9200 for OpenSearch 2.0.1, but however the documentation still says to use Default 9300 port. 

This issue is raised to help OpenSearch Kubernetes Operator compatible with 2.0.0 series of OpenSearch.
https://github.com/Opster/opensearch-k8s-operator/issues/176

--- a/_security-plugin/configuration/security-admin.md
+++ b/_security-plugin/configuration/security-admin.md
@@ -182,12 +182,12 @@ Name | Description
 
 ### OpenSearch settings
 
-If you run a default OpenSearch installation, which listens on transport port 9300 and uses `opensearch` as a cluster name, you can omit the following settings altogether. Otherwise, specify your OpenSearch settings by using the following switches.
+If you run a default OpenSearch installation, which listens on port 9200 and uses `opensearch` as a cluster name, you can omit the following settings altogether. Otherwise, specify your OpenSearch settings by using the following switches.
 
 Name | Description
 :--- | :---
 `-h` | OpenSearch hostname. Default is `localhost`.
-`-p` | OpenSearch port. Default is 9300---not the HTTP port.
+`-p` | OpenSearch port. Default is 9200 - not the HTTP port.
 `-cn` | Cluster name. Default is `opensearch`.
 `-icl` | Ignore cluster name.
 `-sniff` | Sniff cluster nodes. Sniffing detects available nodes using the OpenSearch `_cluster/state` API.

